simplivity/resources/omnistack_clusters.py

```diff
@@ -110,3 +110,23 @@ def __init__(self, connection, resource_client, data):
         self.data = data
         self._connection = connection
         self._client = resource_client
+        self._clusters = OmnistackClusters(self._connection)
+
+    def set_time_zone(self, time_zone, timeout=-1):
+        """ Sets the time zone for a cluster.
+
+        Args:
+            time_zone: The time zone in case-sensitive region/locale format
+                       for example, "America/New_York"
+            timeout : Time out for the request in seconds.
+        Returns:
+            cluster object
+        """
+
+        method_url = "{}/{}/set_time_zone".format(URL, self.data["id"])
+        data = {"time_zone": time_zone}
+        out = self._client.do_post(method_url, data, timeout)[0]
+        cluster = self._clusters.get_all(show_optional_fields=True,
```

Comment:
I don't think we have to get the OmniStack Cluster object again. Can we just refresh the current object data and return True if the set_time_zone operation is successful

Reply:
Updated the code as per the suggestion, added the refresh method to update the omnistack cluster object.